Sources/SourceKitLSP/SemanticTokensLegend+SourceKitLSPLegend.swift

```diff
@@ -22,10 +22,28 @@ extension SemanticTokenTypes {
   }
 }
 
+extension SemanticTokenModifiers {
+  // Argument labels are considered part of the function name, but this modifier
+  // allows themes to style them differently if desired.
+  package static var parameterLabel: Self { Self(rawValue: UInt32(1) << UInt32(SemanticTokenModifiers.all.count)) }
```

Comment:
Could you add this to the definition of `SemanticTokenModifiers` in [swift-tools-protocols](https://github.com/swiftlang/swift-tools-protocols/blob/main/Sources/LanguageServerProtocol/SupportTypes/SemanticTokenModifiers.swift)?